ads/google/doubleclick.js

```diff
@@ -65,15 +65,25 @@ export function doubleclick(global, data) {
     transform: 'translate(-50%, -50%)',
   });
 
-  if (data.useSameDomainRenderingUntilDeprecated != undefined ||
-      data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT);
+  const fileExperimentConfig = {
+    21060540: 'gpt_sf_a.js',
+    21060541: 'gpt_sf_b.js',
+  };
+// Note that reduce will return the first item that matches but it is expected
+// that only one of the experiment ids will be present.
+  const expFilename = data['experimentId'] && data['experimentId'].split(',')
```

Comment:
I don't think this actually works as is.  Instead, I suggest changing to:

let expFilename;
(data['experimentId'] || '').split(',').forEach(val => expFilename = expFilename || fileExperimentConfig[val]);